devices/fields/dac/h5_u5.yaml

```diff
@@ -0,0 +1,17 @@
+_include:
+  - v6.yaml
+
+CR:
+  TSEL?:
+    Swtrig: [0, Software trigger]
+    Tim1Trgo: [1, Timer 1 TRGO event]
+    Tim2Trgo: [2, Timer 2 TRGO event]
+    Tim4Trgo: [3, Timer 4 TRGO event]
+    Tim5Trgo: [4, Timer 5 TRGO event]
+    Tim6Trgo: [5, Timer 6 TRGO event]
+    Tim7Trgo: [6, Timer 7 TRGO event]
+    Tim8Trgo: [7, Timer 8 TRGO event]
+    Tim15Trgo: [8, Timer 15 TRGO event]
+    Lptim1Ch1: [11, LPTIM1 CH1 event]
+    Lptim2Ch1: [12, LPTIM2 CH1 event]
```

Comment:
[RM0456](https://www.st.com/resource/en/reference_manual/rm0456-stm32u5-series-armbased-32bit-mcus-stmicroelectronics.pdf) Table 343 says lptim**3**_ch1 here while [RM0481](https://www.st.com/resource/en/reference_manual/rm0481-stm32h52333xx-stm32h56263xx-and-stm32h573xx-armbased-32bit-mcus-stmicroelectronics.pdf) Table 281 says lptim**2**_ch1

Reply:
Thank you. Missed this due to outdated RM revisions.